ENIGMAsystem/SHELL/Widget_Systems/Win32/dialogs.cpp

```diff
@@ -203,6 +204,21 @@ static INT CALLBACK GetDirectoryAltProc(HWND hwnd, UINT uMsg, LPARAM lp, LPARAM
   return 0;
 }
 
+static wchar_t* string_to_widechar(string str) {
+  // Number of shorts will be <= number of bytes; add one for null terminator
```

Comment:
Thank you for preserving this comment; I forgot it. Also, maybe update these copyright headers, too. Not as important. Also also, it'll probably be good to move this method somewhere common in the future, since all of Win32 is gonna want to be using it.

Reply:
Agreed. d18de1aa95556fe70e01025d87b15770e2db48e2

We can leave the helper for now, it's ok there. TKG is interested in that helper function for some things and he might be willing to do the work to implement it in the rest of our Win32 systems.